microcosm_pubsub/conventions/lifecycle.py

```diff
@@ -0,0 +1,33 @@
+"""
+Enumeration of common lifecycle changes.
+
+"""
+
+
+class LifecycleChange(set):
```

Comment:
Now a *set*, not an *Enum*

Reply:
What's the main benefit here?

Reply:
ah, so `make_media_type` doesn't need to be tied here.

Reply:
And it's extensible.